### PR TITLE
committing alert2 to bracnhx

### PR DESCRIPTION
**Pull request by saurabhmember**
Hi, I commited the alert2, please check.

### CODE REVIEW

File: src/test/java/com/tools/qa/testcases/AlertsPageTest2.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.tools.qa.testcases;
+
+public class AlertsPageTest2 {
```
Comment (saurabhlead) [inline src/test/java/com/tools/qa/testcases/AlertsPageTest2.java:3]: Hi X,
looks ggood, please add more checks

**Review by saurabhlead — COMMENTED** [review]
please check the code

**Comment by saurabhlead** [comment]
hi x,

please check line number 3

**Comment by saurabhmember** [comment]
HI, i have added print

**Comment by saurabhlead** [comment]
your code looks good , please merge

**Pull request merged**